packages/react-liveness/src/components/FaceLivenessDetector/service/utils/liveness.ts

```diff
@@ -326,11 +326,28 @@ function getPupilDistanceAndFaceHeight(face: Face) {
   return { pupilDistance, faceHeight };
 }
 
-export function generateBboxFromLandmarks(
-  face: Face,
-  oval: LivenessOvalDetails,
-  frameHeight: number
-): BoundingBox {
+export function generateBboxFromLandmarks({
+  sessionInformation,
+  face,
+  oval,
+  frameHeight,
+}: {
+  sessionInformation: SessionInformation;
+  face: Face;
+  oval: LivenessOvalDetails;
+  frameHeight: number;
+}): BoundingBox {
+  const challengeConfig =
+    sessionInformation?.Challenge?.FaceMovementAndLightChallenge
+      ?.ChallengeConfig;
+
+  if (!challengeConfig || !challengeConfig.OvalHeightWidthRatio) {
+    throw new Error('Challenge config not returned from session information.');
+  }
+
+  const { OvalHeightWidthRatio } = challengeConfig;
+  const ovalHWRatio = OvalHeightWidthRatio ?? OVAL_HEIGHT_WIDTH_RATIO;
```

Comment:
Since `OvalHeightWidthRatio` is the only value eventually read from `sessionInformation`, we can add `ovalHWRatio`  as a parameter with a default value of `OVAL_HEIGHT_WIDTH_RATIO`?

Reply:
Yeah I think i did this at first but forgot why I changed it back, I think i was thinking in case we needed other values from session information in the future. Will take your suggestion and just update it to session Info if we ever need it later :)